### PR TITLE
fix(tier4_camera_view_rviz_plugin): fix unusedFunction

### PR DESCRIPTION
**Pull request by kobayu858**
## Description
This is a fix based on cppcheck unusedFunction warnings.

```
common/tier4_camera_view_rviz_plugin/src/bird_eye_view_controller.cpp:102:0: style: The function 'handleMouseEvent' is never used. [unusedFunction]
void BirdEyeViewController::handleMouseEvent(rviz_common::ViewportMouseEvent & event)
^

common/tier4_camera_view_rviz_plugin/src/bird_eye_view_controller.cpp:181:0: style: The function 'lookAt' is never used. [unusedFunction]
void BirdEyeViewController::lookAt(const Ogre::Vector3 & point)
^

common/tier4_camera_view_rviz_plugin/src/bird_eye_view_controller.cpp:186:0: style: The function 'onTargetFrameChanged' is never used. [unusedFunction]
void BirdEyeViewController::onTargetFrameChanged(
^

common/tier4_camera_view_rviz_plugin/src/third_person_view_controller.cpp:229:0: style: The function 'updateTargetSceneNode' is never used. [unusedFunction]
void ThirdPersonViewController::updateTargetSceneNode()
^
```
## Related links

**Parent Issue:**

- Link

## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

**Comment by kobayu858** [comment]
I have confirmed that the three functions suppressed in the comments are actually used.
The remaining `onTargetFrameChanged` is probably not used.

**Comment by Owen-Liuyuxuan** [comment]
@kobayu858 

This function is overriding the [onTargetFrameChanged from the base class](https://github.com/ros2/rviz/blob/1d031d285bd8e3294fc24b474b61b960269eb821/rviz_common/include/rviz_common/frame_position_tracking_view_controller.hpp#L102C16-L102C36) which will be [called by the base class](https://github.com/ros2/rviz/blob/1d031d285bd8e3294fc24b474b61b960269eb821/rviz_common/src/rviz_common/frame_position_tracking_view_controller.cpp#L107).

So I suggest suppressing the test on this function too

File: common/tier4_camera_view_rviz_plugin/src/bird_eye_view_controller.cpp [hunk 1]
```diff
@@ -178,20 +179,12 @@ void BirdEyeViewController::update(float dt, float ros_dt)
   updateCamera();
 }
 
+// cppcheck-suppress unusedFunction
 void BirdEyeViewController::lookAt(const Ogre::Vector3 & point)
 {
   setPosition(point - target_scene_node_->getPosition());
 }
 
-void BirdEyeViewController::onTargetFrameChanged(
```
Comment (Owen-Liuyuxuan) [inline common/tier4_camera_view_rviz_plugin/src/bird_eye_view_controller.cpp:186]: This function is used in https://github.com/ros2/rviz/blob/1d031d285bd8e3294fc24b474b61b960269eb821/rviz_common/src/rviz_common/frame_position_tracking_view_controller.cpp#L110.

Please suppress this also

**Review by Owen-Liuyuxuan — CHANGES_REQUESTED** [review]

File: common/tier4_camera_view_rviz_plugin/src/bird_eye_view_controller.cpp (hunk 1 above)
Comment (kobayu858) [reply common/tier4_camera_view_rviz_plugin/src/bird_eye_view_controller.cpp:186]: @Owen-Liuyuxuan 

Thanks for the advice. I have changed it to suppression in the comments.

**Review by Owen-Liuyuxuan — APPROVED** [review]
LGTM! Thanks for your efforts of improving the coding quality.

**Pull request merged**